dist/api/bundler.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+var api = require('jspm/api');
+var whacko = require('whacko');
+var glob = require('glob');
+var fs = require('fs');
+var url = require('url');
+var path = require('path');
+
+var pluginName = 'view';
+var loader = api.Builder().loader;
```

Comment:
Instead of requiring the file as the API.prototype[somefile]

I am creating a getter.

```
Object.defineProperty(API.prototype, name, {
      get: function() {
        return require(p);
      }
    });
    // instead of 
   API.prototype[name] = require(p);
```